config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -65,6 +65,11 @@ spec:
                           type: string
                       type: object
                   type: object
+              oneOf:
+              - required:
+                - value
+              - required:
+                - valueFrom
```

Comment:
Where these manually added? Is there a way to add this to the API spec in a way that the generator manages this?

Reply:
Yes, by hand. There has been other manual changes since the CRD was generated. To the API spec you mean machine type spec? Or, some place else?

Reply:
I mean updating the type spec to use the kubebuilder validation annotations: https://book.kubebuilder.io/beyond_basics/generating_crd.html#validation